### ISSUE TITLE
provisioned_at fails on windows 10/vagrant/virtualbox

### ISSUE THREAD

**Issue by benlumia007** [Varying-Vagrant-Vagrants/VVV#1851]
```
=> default: Running provisioner: initial-setup (shell)...
    default: Running: inline script
    default: mkdir:
    default: cannot create directory ‘/vagrant’
    default: : Permission denied
    default: touch:
    default: cannot touch '/vagrant/provisioned_at'
    default: : No such file or directory
    default: /tmp/vagrant-shell: line 5: /vagrant/provisioned_at: No such file or directory
    default: cp: cannot create regular file '/vagrant': Permission denied
    default: cp: cannot create regular file '/vagrant'
    default: : Permission denied
    default: ln:
    default: failed to create symbolic link '/vagrant/certificates'
    default: : No such file or directory
```

**Comment by benlumia007** [comment]
I also tested MacOS and Linux and it works fine, it copy the provisioned_at to /vagrant and seems fine. but not for windows/virtualbox.

**Comment by benlumia007** [comment]
On Windows it just shows
`log/provision/provisioner-main.log`

on MacOS and Linux
`log/provisioners/20190610-050218/provisioner-main.log`

`/vagrant/vv-custom.yml`

no such file or directory

**Comment by tomjn** [comment]
We need to put sudo in the new tty provisioner

**Comment by tomjn** [comment]
Fixed by #1852

**Comment by benlumia007** [comment]
well that fixed the permission,  i had another issue where it won't mkdir or touch the file, but finding out that it was my problem where i change how it displays the folder for the date and since windows doesn't know how to read `:` it fails to create. 

so we need to make sure that not all syntax are the same.
